packages/expo-media-library/src/next/index.ts

```diff
@@ -28,23 +69,34 @@ export class Album extends ExpoMediaLibraryNext.Album {
     }
     return ExpoMediaLibraryNext.createAlbum(name, assetsRefs, moveAssets);
   }
+  /*
+   * A static function. Deletes multiple albums at once.
+   * @param albums - An array of `Album` instances to delete.
```

Comment:
```suggestion
   * @param albums - An array of {@link Album} instances to delete.
```


There are other places where we are missing links to types as well.